step.yml

```diff
@@ -95,3 +95,11 @@ inputs:
       value_options:
       - "true"
       - "false"
+  - ignore_stack_difference: "false"
+    opts:
+      title: "Ignore stack difference"
+      summary: "Allow to use the same cache across different Bitrise stacks"
```

Comment:
```suggestion
      summary: "Allow to use the same cache across different Bitrise stacks. Use at your own risk, it might cause build problems."
```